objects/_objects.media.scss

```diff
@@ -8,23 +8,15 @@
  */
 
 .o-media {
-  @include inuit-clearfix();
-  display: block;
+  display: flex;
 }
 
   .o-media__img {
-    float: left;
     margin-right: $inuit-global-spacing-unit;
-
-    > img {
-      display: block;
-    }
-
   }
 
   .o-media__body {
-    overflow: hidden;
-    display: block;
+    flex: 1 1 0%;
```

Comment:
Remove unit when value is `0`.

Reply:
I don't want to support old IE but I think it's better if someone want...: https://github.com/philipwalton/flexbugs#flexbug-4

Reply:
I wasn’t aware of this. Can you put a code comment with exactly that link?